### ISSUE TITLE
Only one send newsletter implementation.

### ISSUE THREAD

**Issue by danielsokolowski** [emencia/emencia-django-newsletter#60]
Hi All,

We now have send_newsletter_continuous and send_newsletter in the core, I think this is confusing and complicates code maintainability. What are your thoughts on picking one implementation and making it the default send_newsletter. 
1. send_newsletter - is the original which with a few bug fixes and tweaks works relatively well and uses predictable cron.
2. send_newsletter_continuous - is a new merged implementation as per https://github.com/Fantomas42/emencia-django-newsletter/pull/53#issuecomment-4193230 and from my limited understanding of it it uses threads, is not cron friendly, is persistent --- I have no experience using it however.

Bottom line I am arguing we ought to have just one manage.py command regardless of which approach is used.
